assets/js/src/components/ResponsiveImageSize.jsx

```diff
@@ -116,6 +119,23 @@ class ResponsiveImageSize extends React.Component {
 
 
   /**
+   * Called when image loading failed
+   */
+  onError(e) {
+    if (this.props.fallbackImage) {
+      this.setState({
+        ...this.state,
```

Comment:
Does this do something different than just calling `this.setState({ imagePath: this.props.fallbackImage })` ?